.github/workflows/ci.yml

```diff
@@ -47,8 +47,13 @@ jobs:
         uses: actions/setup-python@v2
         with:
           python-version: ${{ matrix.python-version }}
+      - name: Install Go
+        uses: actions/setup-go@v3
+        with:
+          go-version: 1.x
 
       - name: Run tests
         run: |
+          go install filippo.io/mkcert@latest
```

Comment:
Might want to run this only on 1 version of Python and also 1 OS (ubuntu) for (1) reliability, idk if this is reliable on Windows and (2) CI execution time

Reply:
I think running on every OS is best, that way we test custom certificates in the system store for each OS.

Reply:
Ok 😄 , I'll have to fix Windows then

Reply:
It looks like the error code on Windows is CERT_E_UNTRUSTEDROOT "A certification chain processed correctly but terminated in a root certificate that is not trusted by the trust provider." (based on https://docs.microsoft.com/en-us/windows/win32/api/wincrypt/ns-wincrypt-cert_chain_policy_status)

It's not immediately obvious to me why that would be the case but maybe there's error output from mkcert?

Reply:
it actually looks like the test that failed is not the new test I'm adding. I think I'll re-run it and see if it reproduces

Reply:
I can't trigger a run. @davisagli or @sethmlarson would you mind changing the setting for Actions to "Require approval for first-time contributors who are new to GitHub"?

Reply:
Done!